shared/instance.go

```diff
@@ -364,6 +364,10 @@ func ConfigKeyChecker(key string, instanceType instancetype.Type) (func(value st
 		if strings.HasSuffix(key, ".uuid") {
 			return validate.IsAny, nil
 		}
+
+		if strings.HasSuffix(key, ".ready") {
```

Comment:
Can we use `.last_state.ready` here like `.last_state.vf.parent` above, so we narrow down which keys are allowed.